.yamato/_abv.yml

```diff
@@ -117,6 +115,120 @@ trunk_verification_trunk:
+all_smoke_tests_2020.2:
+    name: All Smoke Tests - 2020.2
+    dependencies:
+      -  path: .yamato/_abv.yml#smoke_test_playmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_editmode_2020.2
+         rerun: on-new-revision
+all_project_ci_2020.2:
+    name: _ABV for SRP repository - 2020.2
+    dependencies:
+      -  path: .yamato/_projectcontext.yml#all_package_ci_project_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-universal.yml#All_Universal_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-shadergraph.yml#All_ShaderGraph_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-hdrp.yml#All_HDRP_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-vfx_hdrp.yml#All_VFX_HDRP_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-vfx_urp.yml#All_VFX_URP_2020.2
+         rerun: on-new-revision
+    triggers:
+        expression: pull_request.target eq "master" AND NOT pull_request.draft AND NOT pull_request.push.changes.all match ["**/*.md", "doc/**/*", "**/Documentation*/**/*", ".github/**/*", "Tools/**/*"]
+all_project_ci_nightly_2020.2:
+    name: _Nightly ABV against 2020.2
+    dependencies:
+      -  path: .yamato/_abv.yml#all_project_ci_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/_templates.yml#all_template_ci_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_playmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_editmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/universal-android-opengles3.yml#Universal_Android_OpenGLES3_Standalone_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/universal-android-vulkan.yml#Universal_Android_Vulkan_Standalone_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/universal-iphone-metal.yml#Universal_iPhone_Metal_Standalone_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/universal-win-dx12.yml#Universal_Win_DX12_Standalone_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/universal-win-dx12.yml#Universal_Win_DX12_editmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/universal-win-dx12.yml#Universal_Win_DX12_playmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/urpupdate-win-dx11.yml#URPUpdate_Win_DX11_editmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/urpupdate-win-dx11.yml#URPUpdate_Win_DX11_playmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/urpupdate_top_asset_store-win-dx11.yml#URPUpdate_Top_Asset_Store_Win_DX11_editmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/urpupdate_top_asset_store-win-dx11.yml#URPUpdate_Top_Asset_Store_Win_DX11_playmode_2020.2
+         rerun: on-new-revision
+    triggers:
+        recurring:
+          -  branch: master
+             frequency: 0 * * ?
```

Comment:
should we ask the Evergreen people to also track this job for failures?

Reply:
Wouldn't the failures and bisections look the same on trunk and on 2020.2 since the C# code is the same for now?

Reply:
good point, but are we certain that the differences between 2020.2 and trunk won't cause differences in failures? (elthough editor pinning should prevent this, but never know)